demos/checkboxradio/radiogroup.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Checkboxradio - Radio Group</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<script src="../../ui/controlgroup.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "input" ).checkboxradio();
+		$( "fieldset" ).controlgroup();
+	});
+	</script>
+</head>
+<body>
+<div class="widget">
+
+	<h2>Radio Group</h2>
+	<fieldset >
+		<legend>Location: </legend>
+		<label for="radio-1">New York</label>
+		<input type="radio" name="radio-1" id="radio-1">
+		<label>Paris
+			<input type="radio" name="radio-1" id="radio-2">
+		</label>
+		<input type="radio" name="radio-1" id="radio-3">
+		<label for="radio-3">London</label>
+	</fieldset>
+</div>
+
+<div class="demo-description">
+<p>Example markup using the controlgroup widget to create a radio group.</p>
```

Comment:
This demo seems unnecessary, we cover the concept in multiple other demos. If we keep it, similar issues as default demo.